crds/stackgres.io_sgpoolingconfigs.yaml

```diff
@@ -32,7 +32,7 @@ spec:
           metadata:
             type: object
           spec:
```

Comment:
those lines were added by `generate` target, unrelated to changes in this PR, but they're so small I decided to leave them